web/styles/app_components.css

```diff
@@ -281,7 +281,7 @@ input::placeholder {
             }
 
             &.disabled {
-                pointer-events: none;
+                cursor: not-allowed;
```

Comment:
We previously showed the default cursor so let's keep it that way for now and change it later if needed.

Reply:
Sure

Reply:
By default I meant setting `cursor: default` here.

Reply:
Got it